server/web.js

```diff
@@ -222,26 +245,55 @@ exports.createFrontend = function createFrontend(config, db) {
       project: req.params.project,
       collabid: req.params.collabid,
       cutoff: req.params.cutoff,
+      visual: req.query.visual,
     });
   });
   
-  app.get('/dashboard/:project/:collabid/m/:milestone/:cutoff?', authenticate, staffonly, function(req, res, next) {
+  // TODO: Get URL with regex but no cutoff to work
+  app.get('/dashboard/:project/:collabid/m/:milestone/:cutoff?/:regexes?', authenticate, staffonly, function(req, res, next) {
     res.render('dashboard/collab', {
       project: req.params.project,
       collabid: req.params.collabid,
       milestone: req.params.milestone,
       cutoff: req.params.cutoff,
+      visual: req.query.visual,
     });
   });
   
   app.get('/baseline/:project/:filepath(*)', authenticate, staffonly, function(req, res, next) {
+    
     db.getBaseline(req.params.project, req.params.filepath, function(err, baseline) {
       if (err) { return res.status(500).send({ code: err.code, message: err.message }); }
       res.type('text/plain');
       res.setHeader('Cache-Control', 'max-age=3600');
       res.send(baseline);
     });
   });
+
+  // Find the given regex in the text, using fuzzy matching
+  // TODO: Better URL?
+  app.get('/regex/:collabid/:regexes/:cutoff?/f/:filepath(*)', authenticate, staffonly,  function(req, res, next) {
```

Comment:
Remove this handler if it is dead code; if there is useful code in the backend that could be used with the fuzzy regex matcher, let's keep that around on a separate branch in case we decide to return to it. Could also keep this code on that branch if it makes sense.